ppr-api/src/ppr_api/models/registration.py

```diff
@@ -102,7 +102,7 @@ class RegistrationTypes(Enum):
     def json(self) -> dict:
         """Return the registration as a json object."""
         registration = {
-            'baseRegistrationNumber': self.base_registration_num,
+            'baseRegistrationNumber': self.base_reg_num,
```

Comment:
i liked the verbose name ...

Reply:
Property renamed to base_registration_num.